46sshd/module-setup.sh

```diff
@@ -65,6 +65,10 @@ install() {
     # include those files
     inst_multiple -o /etc/systemd/network/*
 
+    # Add command to unlock luks volumes to bash history for easier use
+    echo "systemd-tty-ask-password-agent" >> "$initdir/root/.bash_history"
```

Comment:
Nice idea! This even works when bash is invoked as /bin/sh (as it is in an early userspace ssh session).

Reply:
Cherry-picked this with  https://github.com/gsauthof/dracut-sshd/commit/853872061f372208b26230359fa7ee7717ddd9a5